django_app/redbox_app/redbox_core/admin.py

```diff
@@ -4,14 +4,25 @@
 
 
 class UserResource(admin.ModelAdmin):
-    exclude = ["password"]
+    fields = ["email", "is_superuser", "is_staff"]
```

Comment:
this is all we need to know for this app

Reply:
I'd also suggest `last_login` so we can find stale admins

Reply:
9153bac0b660657a3738c9770582a49235b9571a